wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Command.java

```diff
@@ -258,30 +258,14 @@ default ParallelRaceGroup raceWith(Command... parallel) {
    * decorated without issue.
    *
    * @return the decorated command
-   * @deprecated use {@link #endlessly()} instead.
+   * @deprecated Unsupported, per https://github.com/wpilibsuite/allwpilib/pull/4483
```

Comment:
I'm not really a fan of linking to a PR... can the explanation just be written into the documentation?

Reply:
No, not really, not in a way that gives more info.

Reply:
Not all development computers are connected to internet. I think Peter is just asking for this to be in the comment instead of a link. 

> Perpetual/EndlessCommand violates the assumption that execute() doesn't get called after isFinished() returns true -- an assumption that should be valid.
This was unsafe/undefined behavior from the start, and RepeatCommand provides an easy way to achieve similar end results with slightly different (and safe) semantics.

Reply:
That's a very long deprecation message.

Reply:
Longer than usual, but it should render just fine.